spec/integration/date_time_properties_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'spec_helper'
+
+describe ActiveFedora::Base do
+  before do
+    class Foo < ActiveFedora::Base
+      property :date, predicate: ::RDF::DC.date do |index|
+        index.type :date
```

Comment:
I think we should look at the suffix, because there may be date fields that don't declare `index.type :date`  For example this line: https://github.com/projecthydra/active_fedora/blob/master/lib/active_fedora/indexing_service.rb#L31  produces: `"system_create_dtsi"=>"2015-10-23T14:06:19Z"`